src/marketing/index.js

```diff
@@ -0,0 +1,41 @@
+import DataFile from './data-file';
+import MESSAGES from './messages';
+import renderTemplate from '../utils/render-template';
+import WARNING_MESSAGES from '../notifications/warning-message';
+import log from '../cli/log';
+import debug from 'debug';
+
+export const NUMBER_RUNS_BETWEEN_SHOW_MESSAGE = 20;
```

Comment:
So, we discussed and decided to show message on first run(in other words we show it if the DataFile is not exists) for this version and every 10 runs.